docs/conf.py

```diff
@@ -352,4 +352,5 @@
 nitpick_ignore = [
     ("py:mod", "aiohttp"),  # undocumented, no `.. currentmodule:: aiohttp` in docs
     ("py:class", "aiohttp.SimpleCookie"),  # undocumented
+    ("py:class", "RequestHandler"), # undocumented
```

Comment:
Please change this to actual full importable name in the form of `aiohttp...RequestHandler`. Find out what it was supposed to point at and use that. Also, change the references in the document to reflect this full name too.

Reply:
it will be quite hard

Reply:
Just use `aiohttp.web.RequestHandler`